### ISSUE TITLE
Cannot find module './config_schema_v2.1.json'

### ISSUE THREAD

**Issue by pimterry** [resin-io/resin-cli#844]
https://sentry.io/resinio/cli/issues/512307906/

```
Error: Cannot find module './config_schema_v2.1.json'
  File "/snapshot/resin-cli/build/utils/compose.js", line 67, in createProject
    composition = compose.normalize(composition);
  File "/snapshot/resin-cli/build/utils/compose.js", line 104, in null.<anonymous>
    return createProject(projectPath, composeStr, projectName);
...
(14 additional frame(s) were not displayed)

Error: Cannot find module './config_schema_v2.1.json'
```

Looks like a standalone bug - we're not including the schemas used in resin-compose-parse: https://github.com/resin-io-modules/resin-compose-parse/blob/master/src/schemas/config_schema_v2.1.json

**Comment by robsonvn** [comment]
+1

**Comment by pimterry** [comment]
Thanks for the reminder @robsonvn! This is now fixed in v7.3.4.
